package.json

```diff
@@ -5,11 +5,13 @@
   "dependencies": {
     "react": "^16.4.0",
     "react-dom": "^16.4.0",
-    "react-scripts-ts": "2.16.0"
+    "react-router": "^4.3.1",
+    "react-scripts-ts": "2.16.0",
+    "webpack-cli": "^3.1.0"
```

Comment:
~3.0.0 please to match https://github.com/zowe/zlux-app-manager/blob/master/virtual-desktop/package.json

Reply:
oh also, always ~ not ^ due to people in npm not versioning their code carefully enough. ^ can give you breaking changes over time.